corehq/apps/reports/standard/sms.py

```diff
@@ -322,49 +319,123 @@ def uses_locations(self):
 
     @property
     def rows(self):
```

Comment:
is this going to mean that when you download this report, it will give you only the paginated records, or can you still download the entire date range?